crypto/src/vrf/mod.rs

```diff
@@ -87,6 +89,25 @@ impl VRFPrivateKey {
         }
     }
 
+    pub fn new_from_bytes(
```

Comment:
Please rename this to `new_using_random_bytes`, and add documentation above it saying `This function initializes the key using the bytes safely, but DOES NOT mean that the bytes will be used as the key. There could be an internal algorithm that does the initialization. The function expects a random sequence of bytes.`

I can't find a better way to protect from abusing this.